packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -131,11 +143,28 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
   }
 
   Simulation _createFlingSimulation(double velocity) {
-    return scrollBehavior.createFlingScrollSimulation(scrollOffset, velocity);
+    // Assume that we're rendering at atleast 15 FPS. Stop when we're
+    // scrolling less than one logical pixel per frame. We're essentially
+    // normalizing by the devicePixelRatio so that the threshold has the
+    // same effect independent of the device's pixel density.
+    double endVelocity = pixelToScrollOffset(15.0 * ui.window.devicePixelRatio);
```

Comment:
I moved the default tolerance definitions to scrollable_dart so that ScrollBehavior remains unaware of pixels. I didn't (intentionally) change the meaning of the changes from #831.